framework/Supports/FakerPress.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace JustCoded\WP\Framework\Supports;
+
+use JustCoded\WP\Framework\Objects\Singleton;
+
+/**
+ * Class FakerPress
+ * Fakerpress plugin extension which allows to generated faker content for custom fields.
+ */
+class FakerPress {
+	use Singleton;
```

Comment:
you can't use singleton here, because you create several instances for each PostType and singleton can be only one!